catalogue/webapp/pages/search/index.tsx

```diff
@@ -1,21 +1,75 @@
 import { GetServerSideProps } from 'next';
+import { getCookie } from 'cookies-next';
+import { ParsedUrlQuery } from 'querystring';
+
 import { removeUndefinedProps } from '@weco/common/utils/json';
-import { AppErrorProps } from '@weco/common/services/app';
+import { appError, AppErrorProps } from '@weco/common/services/app';
 import { getServerData } from '@weco/common/server-data';
 import Space from '@weco/common/views/components/styled/Space';
 import { NextPageWithLayout } from '@weco/common/views/pages/_app';
-import { getSearchLayout } from 'components/SearchPageLayout/SearchPageLayout';
+import { getSearchLayout } from '@weco/catalogue/components/SearchPageLayout/SearchPageLayout';
 import { Pageview } from '@weco/common/services/conversion/track';
+import { getStories } from '@weco/catalogue/services/prismic/fetch/articles';
+import { Story } from '@weco/catalogue/services/prismic/types/story';
+import { getWorks } from '@weco/catalogue/services/catalogue/works';
+import { getImages } from 'services/catalogue/images';
+import { Image, Work } from '@weco/common/model/catalogue';
+import {
+  decodeQuery,
+  FromCodecMap,
+  stringCodec,
+} from '@weco/common/utils/routes';
+
+// Creating this version of fromQuery for the overview page only
+// No filters or pagination required.
+const codecMap = { query: stringCodec };
+type CodecMapProps = FromCodecMap<typeof codecMap>;
+const fromQuery: (params: ParsedUrlQuery) => CodecMapProps = params => {
+  return decodeQuery<CodecMapProps>(params, codecMap);
+};
 
 type Props = {
+  works: Work[] | undefined;
+  images: Image[] | undefined;
+  stories: Story[] | undefined;
```

Comment:
our convention elsewhere is to use optional properties, i.e.

```suggestion
  works?: Work[];
  images?: Image[];
  stories?: Story[];
```

they are slightly different – `works?: Work[]` means the `works` property is optional, whereas `works: Work[] | undefined` always require something be defined, but I don't think we care about the distinction.

Reply:
Agree to align, I think I played with this as I was creating them and it was needed, but am now doing a big cleanup. Please do check what comes out of it :)